pom.xml

```diff
@@ -115,6 +115,10 @@
     <url>https://builds.apache.org/view/A-D/view/Accumulo/</url>
   </ciManagement>
   <properties>
+    <accumulo.anyClusterTests>org.apache.accumulo.test.categories.AnyClusterTest</accumulo.anyClusterTests>
+    <accumulo.integrationTestsExcludedGroups />
+    <accumulo.integrationTestsGroups>${accumulo.anyClusterTests},${accumulo.miniclusterTests}</accumulo.integrationTestsGroups>
+    <accumulo.miniclusterTests>org.apache.accumulo.test.categories.MiniClusterOnlyTest</accumulo.miniclusterTests>
```

Comment:
Do we need these extra properties? It adds a layer of indirection to the property evaluation, but I'm not sure that indirection adds a lot of value. Under what circumstances would a user prefer to override this property rather than the corresponding groups/excludeGroups properties directly?